### ISSUE TITLE
Accessibility: smiley rating question type the legend is empty and not labelled by aria-labelledby

### ISSUE THREAD

**Issue by andrewtelnov** [surveyjs/survey-library#10788]
Refer to [this issue](https://surveyjs.answerdesk.io/internal/ticket/details/T26024) in the Support Center for more information.
